src/styles/blocks/stars.scss

```
@@ -0,0 +1,24 @@
+@import "../utils/mixins";
+@import "./utils/variables";
+
+.stars {
+  display: flex;
+
+  &__star {
+    @include star($star-size);
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
```

```suggestion
  &__star {
    @include star($star-size);

    background-image: url(../images/star.svg);
    background-repeat: no-repeat;
    background-position: center;
    
    &:not(:last-child) {
      margin-right: 4px;
    }
  }
```
You know what to do ;)